core/src/test/kotlin/fr/sncf/osrd/utils/DummyInfra.kt

```diff
@@ -420,7 +420,7 @@ class DummyInfra : RawInfra, BlockInfra {
     }
 
     override fun getTrackChunkOperationalPointParts(trackChunk: TrackChunkId): StaticIdxList<OperationalPointPart> {
-        TODO("Not yet implemented")
+        return mutableStaticIdxArrayListOf()
```

Comment:
Don't know what the plan is with `DummyInfra`. Might wanna check with a maintainer if this should be really implemented and added to test suite.

Reply:
I talked about that with @eckter.